nj-core/src/lib.rs

```diff
@@ -120,7 +123,3 @@ macro_rules! method {
         nj::core::Property::new($name).method($rs_method)
     }};
 }
-
-pub fn init_logger() {
```

Comment:
unfortunately, this will cause regression because now default logger is not initialized.  Instead, if `node_bindgen_log` is not defined, standard logger will be defaulted, otherwise custom logger will be inited.